### DISCUSSION TITLE
datahub.io down / SSL error?

### DISCUSSION THREAD

**Discussion by marko-knoebl** [datasets/commons#391] [General]
I tried to access datahub.io yesterday and today and it failed - showing me a message about SSL errors in the browser.

Chrome reports ERR_SSL_VERSION_OR_CIPHER_MISMATCH and Firefox reports SSL_ERROR_NO_CYPHER_OVERLAP

Down checkers seem to report the same, e.g.:

https://www.isitdownrightnow.com/datahub.io.html

which says it's been down for 5 days.

I haven't been on the site for a long time - is this a recent problem?

**Comment by marko-knoebl** [comment]
seems to be working again :)

**Reply by anuveyatsu** [reply]
hi @marko-knoebl yes, thanks for flagging .. as you guessed we had some ssl certificate issues that have been resolved.
